packages/instant/src/util/analytics.ts

```diff
@@ -177,4 +180,24 @@ export const analytics = {
         trackingEventFnWithPayload(EventNames.TOKEN_SELECTOR_CHOSE)(payload),
     trackTokenSelectorSearched: (searchText: string) =>
         trackingEventFnWithPayload(EventNames.TOKEN_SELECTOR_SEARCHED)({ searchText }),
+    trackQuoteFetched: (buyQuote: BuyQuote, fetchedVia: QuoteFetchedVia) =>
+        trackingEventFnWithPayload(EventNames.QUOTE_FETCHED)({
+            ...buyQuoteEventProperties(buyQuote),
+            fetchedVia,
+        }),
+    trackQuoteError: (
+        errorMessage: string,
+        assetName: string,
+        assetData: string,
+        assetAmount: BigNumber,
+        fetchedVia: QuoteFetchedVia,
+    ) => {
+        trackingEventFnWithPayload(EventNames.QUOTE_ERROR)({
+            errorMessage,
+            assetName,
```

Comment:
Do we need `assetName` and `assetData` here?

Reply:
ah good call, we should be able to rely on `selectedAsset` event properties here, unless there's a race condition